error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/AssortedRules.java

```diff
@@ -103,8 +103,7 @@ T before(Iterator<T> iterator, T defaultValue) {
     }
 
     @AfterTemplate
-    @Nullable
-    T after(Iterator<T> iterator, T defaultValue) {
+    @Nullable T after(Iterator<T> iterator, T defaultValue) {
       return Iterators.getNext(iterator, defaultValue);
     }
```

Comment:
This reverts the changes in ff2ed6f82cc8cface6f63829fb67eb8cc4520601, as a result of google/google-java-format#869.